src/index.html

```diff
@@ -7,8 +7,77 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" 
+          rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="logo">
+          <img
+            src="images/logo.png"
+            alt="moyo-logo"
+            width="40"
+            height="40"
+          >
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__li">
+              <a 
+                href="#apple" 
+                class="nav__link hover-underline"
+              >Apple</a>
```

Comment:
check the code style according to the checklist
```suggestion
              <a 
                href="#apple" 
                class="nav__link hover-underline"
              >
                Apple
              </a>
```